### PR TITLE
Make Return and Enter key behave identical in markdown editor

### PR DESCRIPTION
**Pull request by fnkabit**
Implements https://github.com/pbek/QOwnNotes/issues/2013

### CODE REVIEW

**Review by Waqar144 — APPROVED** [review]
Looking good!

File: qmarkdowntextedit.cpp [hunk 1]
```diff
@@ -204,12 +204,12 @@ bool QMarkdownTextEdit::eventFilter(QObject *obj, QEvent *event) {
             return bracketClosingCheck(QLatin1Char('{'), QLatin1Char('}'));
         } else if (keyEvent->key() == Qt::Key_BracketRight) {
             return bracketClosingCheck(QLatin1Char('['), QLatin1Char(']'));
-        } else if (keyEvent->key() == Qt::Key_Return &&
+        } else if ( (keyEvent->key() == Qt::Key_Return || keyEvent->key() == Qt::Key_Enter) &&
```
Comment (pbek) [inline qmarkdowntextedit.cpp:207]: Can you please remove your leading space, since there isn't any anywhere else. 😉

File: qmarkdowntextedit.cpp [hunk 2]
```diff
@@ -204,12 +204,12 @@ bool QMarkdownTextEdit::eventFilter(QObject *obj, QEvent *event) {
...
                    keyEvent->modifiers().testFlag(Qt::ShiftModifier)) {
             QTextCursor cursor = this->textCursor();
             cursor.insertText("  \n");
             return true;
-        } else if (keyEvent->key() == Qt::Key_Return &&
+        } else if ( (keyEvent->key() == Qt::Key_Return || keyEvent->key() == Qt::Key_Enter) &&
```
Comment (pbek) [inline qmarkdowntextedit.cpp:212]: Here too: Can you please remove your leading space, since there isn't any anywhere else. 😉

**Comment by pbek** [comment]
Great, thank you!

**Comment by fnkabit** [comment]
Thanks!

**Pull request merged**